docs/contributing/code-style.md

```diff
@@ -59,45 +59,50 @@ The Vagrant setup process runs this for you.
 - Puppet configuration
 - custom checks (e.g. trailing whitespace and spaces-not-tabs)
 
-### Tests
+### Check your code against automated tests
```

Comment:
"Use tests to verify logic you write. This section might want to link to the testing philosophy guide and/or